src/eval.h

```diff
@@ -3,15 +3,17 @@
 
 #define EVAL_MAX_VARIABLE_NAME_LENGTH 300
 
+/** List of `struct rrdvar`*/
 typedef struct eval_variable {
-    char *name;
-    uint32_t hash;
-    struct rrdvar *rrdvar;
-    struct eval_variable *next;
+    char *name;                 ///< Uniqe name
+    uint32_t hash;              ///< Hash of `name`
+    struct rrdvar *rrdvar;      ///< The variable
+    struct eval_variable *next; ///< Next item in the list
 } EVAL_VARIABLE;
 
+/// ktsaou: Your help needed
```

Comment:
This is the internal representation infix expressions used in alarms. netdata splits the expression to a tree, where each node is performing a calculation.